README.rst

```diff
@@ -25,3 +25,16 @@ or you can use it from the configurator:
         '/hello/{name}', view_callable,
         renderer='json'
      )
+
+
+See also
+=========================
```

Comment:
Could we describe that these are influences but not alternatives? May be confusing for someone if they think they can go use Klein instead.